templates/geosafe/analysis/create.html

```diff
@@ -574,7 +574,8 @@ <h4 class="modal-title" id="myModalLabel">{% trans "Upgrade Your Browser" %}</h4
                     }]
             }).addTo(map);
 
-            {# Showing markers for all available layers #}
+            {# Showing markers for all available layers, except if analysis being run #}
+            {% if not is_analysis_running %}
```

Comment:
See comments about `is_analysis_running`. Do not use this.
Use: 
`{% if not analysis or analysis.get_task_state != 'PENDING' %}`
which means:
show markers for all available layers,
if analysis not yet created or
analysis is created but not PENDING.